nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -23,6 +23,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - [Clevis](https://github.com/latchset/clevis), a pluggable framework for automated decryption, used to unlock encrypted devices in initrd. Available as [boot.initrd.clevis.enable](#opt-boot.initrd.clevis.enable).
 
+- [TuxClocker](https://github.com/Lurkki14/tuxclocker), a hardware control and monitoring program. Available as [programs.tuxclocker](#programs.tuxclocker.enable).
```

Comment:
This link to the option is broken, according to the failing manual CI check. I think it needs to be `#opt-programs.tuxclocker.enable`?